### ISSUE TITLE
Wroughtnaut passively damages nearby mobs and players sometimes

### ISSUE THREAD

**Issue by BobMowzie** [BobMowzie/MowziesMobs#196]
This is a new one...
Seen in Direwolf20's MCEternal playthrough. Ruined his encounter :(

**Comment by fkshitanoiyingcrap** [comment]
Is this when he just walks around, no animations, and just hits you constantly, just by you being somewhat near it ?

Cause yeah, I'm getting this problem...

Just being around Wroughtnaut after waking it in its room and it will hurt me without delay just by me getting close to it, its not doing any attacks, just walking around

**Comment by BobMowzie** [comment]
Oh, cool. Because I can't seem to recreate this issue.
Do you have any other mods installed?

**Comment by fkshitanoiyingcrap** [comment]
The only things I can think of that may be causing this in my mods list, could be:

- Epic Siege Mod [https://www.curseforge.com/minecraft/mc-mods/epic-siege-mod](url)

- Scaling Health  [https://www.curseforge.com/minecraft/mc-mods/scaling-health](url)

- Infernal Mobs [https://www.curseforge.com/minecraft/mc-mods/atomicstrykers-infernal-mobs](url)

- Better Combat [https://www.curseforge.com/minecraft/mc-mods/better-combat-rebirth](url)

- Grimoire of Gaia 3 [https://www.curseforge.com/minecraft/mc-mods/grimoire-of-gaia](url)

Most of these I don't even actually see how they could cause this, but out of all the mods I have, only these seem like they could possibly have anything to do with it...

Mostly Better Combat & Epic Siege Mod seem like they could be the issue, idk

This just happened to me a few minutes ago, I left the room and waited a couple minutes until the Wroughtnaut went back to sleep or whatever and fought it again and it was fixed.

**Comment by BobMowzie** [comment]
That's super strange that the problem just fixes itself. It must be very situational...
Ugh, these are the worst kinds of bugs...

**Comment by fkshitanoiyingcrap** [comment]
Yes, I was going to do a quick recording of it, but by the time I thought of it and returned to the Wroughtnaut, it had "fixed" itself & I was able to fight it as it should be and defeat it, versus previously not even being able to get near it because it had some invisible force field of instant attacks while it was just walking around not doing any sort of attack animation.

**Comment by fkshitanoiyingcrap** [comment]
Btw, I would like to thank you for this amazing mod, the animations and the way you interact with the mobs, and they with the world, even in the tiniest way is just amazing for a game like minecraft.

Its a very "rare" mod indeed in terms of quality, haha!

**Comment by BobMowzie** [comment]
Thanks!
Let me know if you see this issue again.

**Comment by Mofo50C** [comment]
i think the mod thats interacting with it is infernal mobs because it sometimes does this for me too and the only mod i have in common with you is infernal mobs i dont think scaling health or better combat has any issues with it and grimoire of gaia just adds mobs. Epic siege im not sure cuz im getting the issue without it.

**Comment by NEFFLEAFF** [comment]
I only have mobBattle mod with mowzie's mobs 1.5.6 and the same error

**Comment by BobMowzie** [comment]
That’s really helpful to know! Does it happen only after using one of the wands on the Wroughtnaut? Or does it happen by itself anyway?

**Comment by NEFFLEAFF** [comment]
Only happens when the wroughnaught fight mobs.
Inclusive fighting the friendly barakoas.

**Comment by NEFFLEAFF** [comment]
![2020-08-07_17 52 20](https://user-images.githubusercontent.com/69360168/89694701-1c255b80-d8d7-11ea-90d1-6ea516e60f1d.png)
![2020-08-07_17 52 39](https://user-images.githubusercontent.com/69360168/89694706-20ea0f80-d8d7-11ea-9edb-1c18d6b75d45.png)
Here can see that the wroughtnaut deals 2 damage to one barakoa
